pageserver/src/context.rs

```diff
@@ -197,6 +206,19 @@ impl Scope {
     }
 }
 
+impl std::fmt::Debug for RequestContext {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("RequestContext")
+            .field("task_kind", &self.task_kind)
+            .field("download_behavior", &self.download_behavior)
+            .field("access_stats_behavior", &self.access_stats_behavior)
+            .field("page_content_kind", &self.page_content_kind)
+            .field("read_path_debug", &self.read_path_debug)
+            // scope, perf_span and perf_span_dispatch are omitted on purpose
+            .finish()
+    }
+}
```

Comment:
Do we actually need the debug impl anywhere?

Reply:
Must have added it because it was derived at some point. Maybe it was removed in some rebase. I'll check.

Reply:
Removed in https://github.com/neondatabase/neon/pull/11140/commits/eed653ea8006703d908d7306ffacbfe9e8526141